### ISSUE TITLE
Setter for length in Implementing_a_custom_Kernel example does not work

### ISSUE THREAD

**Issue by mboedigh** [cornellius-gp/gpytorch#1454]
# 📚 Setter for length in Implementing_a_custom_Kernel does not work

In Implementing_a_custom_Kernel.ipynb the setter for length does not seem to work.
Line 257: 
```    def _set_length(self, value):
        if not torch.is_tensor(value):
            value = torch.as_tensor(value).to(self.raw_length)
        # when setting the paramater, transform the actual value to a raw one by applying the inverse transform
        self.initialize(raw_length=self.raw_length.constraint.inverse_transform(value))
```
should this be? :
```

    def _set_length(self, value):
        if not torch.is_tensor(value):
            value = torch.as_tensor(value).to(self.raw_length)
        # when setting the paramater, transform the actual value to a raw one by applying the inverse transform
        self.initialize(raw_length=self.raw_length_constraint.inverse_transform(value))
```

- Link to [GPyTorch documentation](https://gpytorch.readthedocs.io)
- Link to [GPyTorch examples](https://github.com/cornellius-gp/gpytorch/tree/master/examples)

**Comment by arthus701** [comment]
Oh yes you're absolutely right! That's a typo on my side.

**Comment by gpleiss** [comment]
Closed by #1460
